vscode/src/extension.ts

```diff
@@ -21,7 +21,8 @@ const MIN_TOIT_VERSION = "1.7.0";
 
 async function checkToitCLI(ctx: Context): Promise<boolean> {
   try {
-    const version = clean(cp.execFileSync(ctx.toitExec, [ "version", "-o", "short" ]));
+    const v = cp.execFileSync(ctx.toitExec, [ "version", "-o", "short" ]);
```

Comment:
You could just use the option `encoding 'utf-8'
```suggestion
    const v = cp.execFileSync(ctx.toitExec, [ "version", "-o", "short" ], { encoding: 'utf-8' });
```

If you keep the current approach, explain why it's not a string. It's enough if it's clear through the variable name:
For example:
```suggestion
    const versionBuffer = cp.execFileSync(ctx.toitExec, [ "version", "-o", "short" ]);
```

Reply:
Used the first suggestion. Tested it with `utf8`, so I will keep that.